src/sentry/monitors/endpoints/organization_monitor_details.py

```diff
@@ -131,9 +131,6 @@ def put(self, request: Request, organization, project, monitor) -> Response:
             params["slug"] = result["slug"]
         if "status" in result:
             params["status"] = result["status"]
-
-            # TODO(epurkhiser): Remove dual-writing once status no longer tracks muted
-            params["is_muted"] = result["status"] == MonitorObjectStatus.MUTED
         if "is_muted" in result:
```

Comment:
We're no longer handling the dual write translation here, since setting a `muted` status will no longer do anything.

See the validator for how this works now